src/ppx/ppx_lwt.ml

```diff
@@ -341,9 +341,13 @@ let mapper =
           if !debug then
             [%expr Lwt.backtrace_bind (fun exn -> try raise exn with exn -> exn)
                                       [%e lhs]
-                                      (fun [%p pat] -> [%e rhs])]
+                                      (fun [%p pat] -> [%e rhs])
+              [@ocaml.ppwarning "The operator >> is deprecated"]
+            ]
           else
-            [%expr Lwt.bind [%e lhs] (fun [%p pat] -> [%e rhs])]
+            [%expr (Lwt.bind [%e lhs] (fun [%p pat] -> [%e rhs]))
+              [@ocaml.ppwarning "The operator >> is deprecated"]
+            ]
```

Comment:
You don't provide any location for the attribute here. The location is important, since it's used in the error message. You should build up the attribute by hand using `attribute_of_warning`, it makes things simpler.